ext/gvl_tracing_native_extension/gvl_tracing.c

```diff
@@ -195,14 +211,16 @@ static void render_event(const char *event_name) {
   // Important note: We've observed some rendering issues in perfetto if the tid or pid are numbers that are "too big",
   // see https://github.com/ivoanjo/gvl-tracing/pull/4#issuecomment-1196463364 for an example.
 
+  if (strcmp(event_name, "started_tracing") != 0) {
+    fprintf(output_file,
+      // Finish previous duration
+      "  {\"ph\": \"E\", \"pid\": %"PRId64", \"tid\": %"PRIu64", \"ts\": %f, \"name\": \"%s\"},\n",
+      process_id, thread_id, now_microseconds, previous_event_name
```

Comment:
So if I'm getting this right, the firefox parser wants the event name to be included in both the begin and end phases of a duration event? That's.... kinda weird but ok :man_shrugging: